salad-schema/catalogue.yml

```diff
@@ -0,0 +1,89 @@
+$base: "http://purl.org/ejp-rd/metadata-model/v1/"
```

Comment:
@rajaram yes. There is a python lib called 'Schema salad' that can be used as a command line tool or imported as a Python module, see: https://github.com/common-workflow-language/schema_salad . It can be installed by cloning the repo or through pip: https://pypi.org/project/schema-salad/ . There are several functionalities like validating the schema file. To convert a document to JSON-LD from the command line: `$ schema-salad-tool --print-pre myschema.yml mydocument.yml > mydocument.jsonld`

Reply:
Hi @NuriaQueralt it  looks good to me. Maybe we should play with the tools mentioned above to see the conversion. Please feel free to merge the PR